nubis/puppet/00_nat.pp

```diff
@@ -26,3 +26,7 @@
     nat_in_interface  => 'eth1',
     nat_out_interface => 'eth0',
 }
+
+class { 'nsm':
+    nsm_mailto  => 'michal@mozilla.com'
```

Comment:
This was the value that was in the original config file that I got from them, that being said its actually a little redundant since the bro package has a cron job that basically pulls this file from an s3 bucket thats maintained by infosec. I created this template to future proof it, however what you are saying makes sense and I can probably make that happen